sui/multisig.js

```diff
@@ -58,7 +58,11 @@ async function initMultisigConfig(chain, options) {
     printInfo(JSON.stringify(chain.multisig, null, 2));
 }
 
-async function signTx(keypair, client, options) {
+async function signTx(chain, options) {
+    const [keypair, client] = getWallet(chain, options);
+
+    printInfo('Wallet Address', keypair.toSuiAddress());
```

Comment:
There's a printWalletInfo helper. It can be made to work on offline mode if it doesn't support it